cmd/gitops/beta/run/cmd.go

```diff
@@ -283,18 +287,108 @@ func installFlux(log logger.Logger, kubeClient *kube.KubeHTTP) (fluxVersion stri
 				fluxexec.Timeout(flags.Timeout),
 			),
 		); err != nil {
-			return fluxVersion, err
+			return "", false, err
 		}
 
 		fluxVersion = flags.FluxVersion
 	} else {
 		log.Successf("Flux version %s is found", fluxVersion)
 	}
 
-	return fluxVersion, nil
+	return fluxVersion, true, nil
```

Comment:
This doesn't look right - the else branch needs to return justInstalled = false, doesn't it?